src/item.cpp

```diff
@@ -2592,33 +2592,134 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
         std::string format;
         if( has_flag( flag_FIT ) ) {
             format = _( "<num> <info>(fits)</info>" );
-        } else if( has_flag( flag_VARSIZE ) && encumbrance ) {
+        } else if( has_flag( flag_VARSIZE ) ) {
             format = _( "<num> <bad>(poor fit)</bad>" );
         }
 
         //If we have the wrong size, we do not fit so alert the player
-        if( sizing_level == sizing::human_sized_small_char )  {
+        if( sizing_level == sizing::human_sized_small_char ) {
             format = _( "<num> <bad>(too big)</bad>" );
         } else if( sizing_level == sizing::big_sized_small_char ) {
             format = _( "<num> <bad>(huge!)</bad>" );
         } else if( sizing_level == sizing::small_sized_human_char ||
-                   sizing_level == sizing::human_sized_big_char )  {
+                   sizing_level == sizing::human_sized_big_char ) {
             format = _( "<num> <bad>(too small)</bad>" );
-        } else if( sizing_level == sizing::small_sized_big_char )  {
+        } else if( sizing_level == sizing::small_sized_big_char ) {
             format = _( "<num> <bad>(tiny!)</bad>" );
         }
 
-        info.push_back( iteminfo( "ARMOR", _( "<bold>Encumbrance</bold>: " ), format,
-                                  iteminfo::no_newline | iteminfo::lower_is_better,
-                                  encumbrance ) );
+        info.push_back( iteminfo( "ARMOR", _( "<bold>Encumbrance</bold>:" ), format,
+                                  iteminfo::lower_is_better ) );
```

Comment:
It produces the results you see in the screenshots. Do you know of a better way to output this data? I don't really know how iteminfo works, so there might be a much nicer way. I did have some troubles with making the UI actually look nice and such, so any suggestions are welcome.

Reply:
It can be simpler; just
```
iteminfo( "ARMOR", _( "<bold>Encumbrance</bold>: " ) + format )
```
would produce the same result.  And you can remove `<num>` from `format` and rename it since it's no longer a format thing.